src/component/Header/Logo.tsx

```diff
@@ -0,0 +1,10 @@
+import React, {FC} from 'react';
+import {NavLink} from "react-router-dom";
+export const Logo: FC = () => {
+    let logo = "/images/logo.png";
+    return (
+        <div className="header-logo">

```

Comment:
this is extra div , we have a tag so we can shift this class on a tag.

Reply:
removed extra div in logo component.